Services/UI/OJS.Services.Ui.Business/Implementations/ContestsBusinessService.cs

```diff
@@ -711,5 +725,39 @@ private async Task<ContestFiltersServiceModel> GetNestedFilterCategoriesIfAny(Co
                 official,
                 isUserAdminOrLecturerInContest);
         }
+
+        private (bool isActiveParticipantInCompete, bool isActiveParticipantInPractice) GetParticipantsActivity(
+            Participant? competeParticipant,
+            Participant? practiceParticipant,
+            ContestDetailsServiceModel contestDetailsServiceModel)
+        {
+            if (competeParticipant != null)
+            {
+                competeParticipant.Contest = new Contest
```

Comment:
Doing new on entities is not a good idea in case someone calls SaveChanges down the line.

Reply:
Done.